routes/[version]/datasources/[id]/filters.js

```diff
@@ -2,14 +2,18 @@ const { StatusCodes } = require('http-status-codes');
 const { checkAuth } = require('../../../../middlewares/auth');
 const { getUserCollections } = require('../../../../middlewares/db');
 const { delsertDataSourceCollections } = require('../../../../services/db');
+const { getAndPrepareCollections } = require('../../collections/_helpers');
 
 module.exports = {
     patch: [
         checkAuth,
         patch,
         getUserCollections,
-        (req, res) =>
-            res.sse.sendTo({ reqId: req.id, data: req.session.collections }, 'collections'),
+        async (req, res) => {
```

Comment:
@Valexr `async` in the exports, I don't see it often, sure?

Reply:
Ok. I’m move this to separated function.